data/json/hobbies.json

```diff
@@ -91,5 +91,233 @@
+    "id": "cooking",
+    "name": "Cooking",
+    "description": "You enjoyed cooking for yourself and others.",
+    "points": 1,
+    "skills": [ { "level": 3, "name": "cooking" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "paintball",
+    "name": "Paintball",
+    "description": "Shooting your enemies has always been fun.  Now you can do it for real.",
+    "points": 1,
+    "skills": [ { "level": 3, "name": "rifle" }, { "level": 1, "name": "gun" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "paintball",
+    "name": "Paintball",
+    "description": "Shooting your enemies has always been fun.  Now you can do it for real.",
+    "points": 1,
+    "skills": [ { "level": 3, "name": "rifle" }, { "level": 1, "name": "gun" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "plinking",
+    "name": "Plinking",
+    "description": "You enjoyed plinking with your airsoft pistol.",
+    "points": 1,
+    "skills": [ { "level": 3, "name": "pistol" }, { "level": 1, "name": "gun" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "skeet_shooting",
+    "name": "Skeet Shooting",
+    "description": "Where the skeet go, your bullet follows.  Or atleast, one of them will.",
+    "points": 1,
+    "skills": [ { "level": 3, "name": "shotgun" }, { "level": 1, "name": "gun" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "socializing",
+    "name": "Socializing",
+    "description": "You were an incredibly social individual.",
+    "points": 1,
+    "skills": [ { "level": 3, "name": "speech" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "street_fighting",
+    "name": "Street Fighting",
+    "description": "You were always instigating fights in the street.  Atleast this might come in useful now.",
+    "points": 1,
+    "skills": [ { "level": 1, "name": "melee" }, { "level": 3, "name": "unarmed" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "knife_throwing",
+    "name": "Knife Throwing",
+    "description": "Throwing knives are fun and deadly in the right hands.",
+    "points": 1,
+    "skills": [ { "level": 3, "name": "throw" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "sewing",
+    "name": "Sewing",
+    "description": "You used to sew torn clothes back together and even made your own clothes a few times.",
+    "points": 1,
+    "skills": [ { "level": 3, "name": "tailor" }, { "level": 1, "name": "fabrication" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "knitting",
+    "name": "Knitting",
+    "description": "Maybe knitting tiny sweaters for your neighbor's cat will come in useful.",
+    "points": 1,
+    "skills": [ { "level": 2, "name": "tailor" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "fencing",
+    "name": "Fencing",
```

Comment:
Shouldn't this also give the fencing MA(which has a trait form for Competitive Fencer)?

Reply:
Added :)